doc/overviews/development.rst

```diff
@@ -55,3 +55,36 @@ implementation that is used, tests can be decorated with the following two marke
 
 If you provide the fixture (i.e., an argument to the test) with name ``default_implementation``, then the test will be
 parameterized to test both implementations.
+
+Useful Snippets
+---------------
+
+GPU Leak checker
+~~~~~~~~~~~~~~~~
+Put this code in ``tests/conf.py``::
+
+    import xml.etree.ElementTree as ET
+    import subprocess
+    import shlex
+
+    import torch
+    # initialize torch cuda context
+    a = torch.ones(1, 1).cuda()
+
+
+    def _get_gpu_mem_usage():
+        result = subprocess.check_output(shlex.split("nvidia-smi -x -q"))
```

Comment:
you could get the cuda free memory using the cuda driver API (`cuMemGetInfo`) but no need for that now.